rts/Rendering/FarTextureHandler.cpp

```diff
@@ -90,7 +90,7 @@ int2 CFarTextureHandler::GetTextureCoordsInt(const int farTextureNum, const int
 
 	const int row = texnum       / (texSize.x / iconSize.x);
 	const int col = texnum - row * (texSize.x / iconSize.x);
-	return int2(col, row);
+	return { col, row };
```

Comment:
wonder how you feel about this kind of style

Reply:
use it all the time, minus the leading/trailing spaces